### PR TITLE
Added support for creating an internal load balancer

### PR DESCRIPTION
**Pull request by sbp-mbaan**

### CODE REVIEW

File: lb.tf [hunk 1]
```diff
@@ -41,6 +41,7 @@ resource "aws_security_group" "lb" {
 resource "aws_lb" "default" {
   count              = local.load_balancer_count
   name               = var.name
+  internal           = var.load_balancer_internal
   load_balancer_type = var.protocol == "TCP" ? "network" : "application"
   subnets            = var.public_subnet_ids
```
Comment (borisroman) [inline lb.tf:46]: this one?

Comment (sbp-mbaan) [reply lb.tf:46]: Forgot to save the file.. 🤔

**Review by svanharmelen — CHANGES_REQUESTED** [review]
In my opinion this breaks the existing behavior.

File: variables.tf [hunk 2]
```diff
@@ -27,6 +27,11 @@ variable "desired_count" {
   description = "Desired number of docker containers to run"
 }
 
+variable "ecs_subnet_ids" {
+  type        = list(string)
+  description = "List of subnet IDs assigned to ESC cluster"
```
Comment (svanharmelen) [inline variables.tf:32]:
```suggestion
  description = "List of subnet IDs assigned to ECS cluster"
```

**Review by svanharmelen — APPROVED** [review]

**Review by borisroman — APPROVED** [review]

File: variables.tf [hunk 3]
```diff
@@ -61,6 +66,18 @@ variable "image" {
   description = "Docker image to run in the ECS cluster"
 }
 
+variable "load_balancer_internal" {
```
Comment (TommyM) [inline variables.tf:69]: I also like load_balander better. But there was already a variable called loadbalancer_eip it seems. The difference is really triggering 😂

**Pull request merged**